tests/test_train_lm.py

```diff
@@ -9,10 +9,12 @@
 import tiny_test_corpus
 from levanter.distributed import RayConfig
 from levanter.logging import WandbConfig
+from levanter.utils.py_utils import logical_cpu_core_count
 
 
 def setup_module(module):
-    ray.init("local", num_cpus=10)
+    ray_designated_cores = max(1, logical_cpu_core_count() - 2)
```

Comment:
could you actually remove the -2 from these? the idea is to leave a bit of room just in case

Reply:
Sure. I'll remove the -2.

Reply:
======== 67 passed, 18 skipped, 863 warnings in 499.79s (0:08:19) =======

Reply:
Also, parallel tests run 43% faster with 2 workers, wherein pytest-xdist was used to run 2 worker threads.

➜  levanter git:(sp-ray-cores) ✗ pytest -n 2 tests
================= 67 passed, 18 skipped, 867 warnings in 348.80s (0:05:48) ================